spec/system/purchasing_a_reservation_on_behalf_of_spec.rb

```diff
@@ -72,6 +72,20 @@
     end
   end
 
+  describe "when the user is not authorized" do
+    before do
+      instrument.update!(requires_approval: true)
+    end
+
+    it "can purchase" do
+      click_link instrument.name
+      expect(page).to have_content("The user you are ordering for is not on the authorized list for this instrument.")
+      select user.accounts.first.description, from: "Payment Source"
+
+      click_button "Create"
```

Comment:
Is there another expectation after this?

Reply:
Oops. There should be.